package.json

```diff
@@ -48,6 +47,7 @@
     "mocha-lcov-reporter": "^1.3.0",
     "mocha": "^5.2.0",
     "nock": "^9.4.1",
+    "nyc": "^11.6.0",
```

Comment:
I think we can also move: babel-eslint, chai, and eslint to dev. 
How about jade and morgan -- I don't see those in use at all. Can they be removed?

Reply:
moved and removed.